src/components/AWSDeploy.vue

```diff
@@ -37,7 +37,7 @@
       <div class="quickstart-container mobile-view">
         <a v-if="databaseTab === 'Terraform'" target="_blank" rel="noopener" id="macos-quickstart-link" href="https://github.com/yugabyte/terraform-aws-yugabyte">Complete Docs</a>
         <a v-else-if="databaseTab === 'EKS'" target="_blank" rel="noopener" id="macos-quickstart-link" href="https://docs.yugabyte.com/preview/deploy/kubernetes/multi-zone/eks/helm-chart/">Complete Docs</a>
-        <a v-else target="_blank" rel="noopener" id="macos-quickstart-link" href="https://docs.yugabyte.com/preview/deploy/public-clouds/aws/">Complete Docs</a>
+        <a v-else target="_blank" rel="noopener" id="macos-quickstart-link" href="https://docs.yugabyte.com/preview/deploy/public-clouds/aws/cloudformation/">Complete Docs</a>
```

Comment:
Another observation related to [Terraform](https://deploy-preview-73--upbeat-panini-d4c778.netlify.app/cloud#aws) (I know you haven't changed this file in this PR , but wanted to point out anyway)- The _complete docs_ link for Terraform is pointing to its GitHub repo: https://github.com/yugabyte/terraform-aws-yugabyte and not to the docs link - https://docs.yugabyte.com/preview/deploy/public-clouds/aws/terraform/. This is true for AWS, GCP and Azure.

Reply:
thanks! i think i caught them all now...